doc/rpc/pathfinder_rpc_api.json

```diff
@@ -141,6 +141,33 @@
                     "$ref": "#/components/schemas/TX_GATEWAY_STATUS"
                 }
             }
+        },
+        {
+            "name": "pathfinder_subscribe_newHeads",
```

Comment:
Does the [snip](https://github.com/matthieuauger/SNIPs/blob/main/SNIPS/snip-5.md#example-1) require updating?

I.e. the snip specifies generic `pathfinder_subscribe` where the first param is the subscription type. However in the code we have a dedicated subscription method instead.

Reply:
I think for now we are doing this as is with just `newHeads` then getting feedback from users after they have had a chance to use it. But @matthieuauger can elaborate better?

Reply:
I see two topics here: 
- having separate subscription methods (xxx_susbcribe_newHeads) over a single one with params (as in ethereum): when we discussed this topic, it seemed it would be easier in terms of user experience and maintenance not to copy ethereum and chose separate method instead, so SNIP will definitely need to be updated and as @shramee said we will adapt SNIP/implementation if we find utility having a single method during our user feedbacks
- having pathfinder_subscribe_event over starknet_subscribe_event: I didn't see all pull request feedbacks, did we change this recently? I initially chose starknet_subscribe to have a generic subscribtion method not coupled to pathfinder if other starknet clients want to implement such mechanism, I'd prefer using starknet_subscribe_event but I may miss things there

Reply:
Using `starknet_xxx` would imply that it is a standardized starknet RPC method - which it currently is not.

Once these get added to the starknet RPC specification then we can change it to `starknet_xxx`.

Reply:
Ok sounds good!